keyboards/ckeys/handwire_101/keymaps/default/config.h

```diff
@@ -0,0 +1,6 @@
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "../../config.h"
```

Comment:
This line is redundant; the QMK build script automatically includes all the `config.h` files in the keyboard's build tree.

```suggestion
```